lib/loader.c

```diff
@@ -128,7 +128,9 @@ static GLboolean supportsSRGB = GL_TRUE;
  * @brief workaround mismatch of glGetString declaration and standard string
  *        function parameters.
  */
-#define glGetString(x) (const char*)glGetString(x)
+#ifndef KTX_USE_GL3W_GETPROC
```

Comment:
Why is this #ifdef needed? Does GLEW have the same define? Or is it just unnecessary because the compiler you are using makes char unsigned? If the latter there is no need to #ifdef it. The define is here because gl.h defines this as returning GLubyte but most compilers make char signed.